### ISSUE TITLE
Eq. 4 in the paper.

### ISSUE THREAD

**Issue by omiita** [lucidrains/vit-pytorch#3]
Thank you for creating this repo! It's so beneficial for us!
I'm wondering about Eq.4( i.e. MLP Head) in their paper and your implementation.

In their paper,  Eq.4 is written in:

<img width="996" alt="Screen Shot 2020-10-11 at 3 29 03" src="https://user-images.githubusercontent.com/29161998/95662437-eb310380-0b71-11eb-9b3f-35c98b91c2a4.png">

This equation is supposed to indicate MLP Head, but, clearly this is not MLP but Layer Norm. **Is this typo? Or, am I missing something?**

And, in [your implementation of MLP Head](https://github.com/lucidrains/vit-pytorch/blob/main/vit_pytorch/vit_pytorch.py#L92), it seems just FFN(except output shape). So, **based on what part of the paper did you implement it(apparently not Eq.4)?**

In short, my questions are:

1. **Is Eq.4 typo? Or, am I missing something?**
2. **Based on what part of the paper did you implement [MLP Head](https://github.com/lucidrains/vit-pytorch/blob/main/vit_pytorch/vit_pytorch.py#L92)?**

Again, thank you for such a quick work.  Looking forward to your reply!

**Comment by lucidrains** [comment]
@omiita Hello! The MLP head is a standard adapter we NLP people often tack onto the end of an attention encoder. They come in all variations, and really isn't all that important. Some people like to layer norm before doing a linear to logits, some people do not. I'll change it so it reflects the equation in the paper.

**Comment by lucidrains** [comment]
https://github.com/lucidrains/vit-pytorch/releases/tag/0.2.0

**Comment by omiita** [comment]
>  Some people like to layer norm before doing a linear to logits, some people do not.

I see. Thank you so much!
